vultr/resource_instance.go

```diff
@@ -224,6 +230,14 @@ func resourceInstanceRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("tag", instance.Tag)
 	d.Set("vcpus", instance.VCpus)
 
+	if instance.V6Networks != nil {
```

Comment:
There is no more need for this nil check;  we can range over the ipv6s happily and if the slice is nil and there is nothing to loop over then we set the property to an empty slice.

Reply:
You're right, thanks. My Go really has become a bit rusty...